docs/getting_started.md

```diff
@@ -218,76 +218,93 @@ Here are some resources that can be used to get you started working with Seurat
-Quality-control statistics calculated by [`DropletUtils::CleanTagCounts()`](https://rdrr.io/github/MarioniLab/DropletUtils/man/cleanTagCounts.html) are provided in the `colData` slot of the `altExp` (`colData(altExp(filtered_sce))`), as described in {ref}`Additional SingleCellExperiment components for CITE-seq libraries (with ADT tags) <sce_file_contents:Additional SingleCellExperiment components for CITE-seq libraries (with ADT tags)>`.
-We recommend filtering out these low-quality cells before proceeding with ADT normalization and downstream analyses.
+For `SingleCellExperiment` objects, quality-control statistics calculated by [`DropletUtils::CleanTagCounts()`](https://rdrr.io/github/MarioniLab/DropletUtils/man/cleanTagCounts.html) are provided in the `colData` slot of the `altExp` (`colData(altExp(filtered_sce))`) as described in {ref}`Additional SingleCellExperiment components for CITE-seq libraries (with ADT tags) <sce_file_contents:Additional SingleCellExperiment components for CITE-seq libraries (with ADT tags)>`.
+
+For `AnnData` objects, these same quality-control statistics are provided in the `obs` slot of the `AnnData` object as described in {ref}`Additional AnnData components for CITE-seq libraries (with ADT tags) <sce_file_contents: Additional AnnData components for CITE-seq libraries (with ADT tags)>`.
 
+We recommend filtering out these low-quality cells before proceeding with ADT normalization and downstream analyses.
 
 Here are some additional resources that can be used for working with ADT counts from CITE-seq experiments:
 - [Integrating with Protein Abundance, Orchestrating Single Cell Analysis](http://bioconductor.org/books/3.15/OSCA.advanced/integrating-with-protein-abundance.html)
 - [Seurat vignette on using with multimodal data](https://satijalab.org/seurat/articles/multimodal_vignette.html)
 
 
-
 ## Special considerations for multiplexed samples
 
 If the dataset that you have downloaded contains samples that were multiplexed (i.e. cells from multiple samples have been combined together into one library), then the steps to take after downloading will be a little different than for libraries that only contain cells corresponding to a single sample.
 Here, multiplexed samples refer to samples that have been combined together into one library using cell hashing and then sequenced together.
 This means that a single library contains cells or nuclei that correspond to multiple samples.
 Each sample has been tagged with a hashtag oligo (HTO) prior to mixing, and that HTO can be used to identify which cells belong to which sample within a multiplexed library.
 The libraries available for download on the portal have not been separated by sample (i.e. demultiplexed), and therefore contain data from multiple samples.
+Note that multiplexed sample libraries are only available as `SingleCellExperiment` objects, and are not currently available as `AnnData` objects.
```

Comment:
I'm hesitant to put in anything about working with AnnData here because they will have to pull out the HTO data as a separate AnnData object, so it's more complex than just converting it. I think the focus here is to just to tell them where the information is in the object to demultiplex. We could include a line at the end that if they would like to convert the demultiplexed libraries to AnnData they can use `zellkonverter`.

Reply:
I agree with your changes! I'd still add this new line though so that these two sentences (this and the next one you've added about zellkonverter) are 1 separate paragraph together.